.github/workflows/unit-tests.yaml

```diff
@@ -0,0 +1,32 @@
+name: tests-unit
+run-name: Tests (unit)
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+env:
+  TELEPORT_GET_VERSION: v10.2.0
+  TELEPORT_ENTERPRISE_LICENSE: ${{ secrets.TELEPORT_ENTERPRISE_LICENSE }}
+
+jobs:
+  test:
+    name: Tests (unit)
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Teleport Plugins
+        uses: actions/checkout@v3
+
+      - name: Setup Go 1.19.3
+        uses: actions/setup-go@v3
+        with:
+          go-version: '1.19.3'
+
+      - name: make test-tooling
+        run: make test-tooling
+
+      - name: go test lib
```

Comment:
No make target to cover this ?

Reply:
> I found out that Makefile make test does not run either lib/* tests or specific plugins tests. I guess, this needs to be addressed in a separate PR.

Just noticed that, why make it a separate PR, I'd lump it in here to make sure it doesn't get forgotten

Reply:
Okay. Introduced test-unit target.